assembly/src/parser/grammar.lalrpop

```diff
@@ -743,6 +744,12 @@ Debug: Instruction = {
             None => Instruction::Debug(DebugOptions::LocalAll),
         }
     },
+    "debug" "." "adv_stack" <n:MaybeImm<U8>> => {
+        match n {
+            Some(n) => Instruction::Debug(DebugOptions::AdvStackTop(n.map(|spanned| spanned.into_inner()))),
+            None => Instruction::Debug(DebugOptions::AdvStackTop(1.into())),
```

Comment:
What is the meaning of `1`?

Reply:
Ah never mind, this is a common path for printing either the top or the `n` elements, right?